Guides/Python/Django.md

```diff
@@ -60,13 +60,60 @@ Left from the colon we specified the **required** process type called `web`
 followed by the command that starts the app and listens on the port specified
 by the environment variable `$PORT`.
 
+### The Actual Application Code
```

Comment:
I don't think we need to explain the Django specifics here. This is the official Django tutorial app. You can link to the official tutorial there for details. Simply state that it's the same and looks something like this, followed by the code example.